### DISCUSSION TITLE
helm部署archery pod的状态一直Init

### DISCUSSION THREAD

**Discussion by cslingjun** [hhyo/Archery#2611] [Q&A] [answered]
操作步骤如下图
<img width="1066" alt="image" src="https://github.com/hhyo/Archery/assets/24772668/a39447ce-3fe3-41f0-b956-79b603f23a71">
查看pod信息
<img width="781" alt="image" src="https://github.com/hhyo/Archery/assets/24772668/9d50015a-7735-4f19-bf82-655e2318ffec">
查看启动事件
<img width="1196" alt="image" src="https://github.com/hhyo/Archery/assets/24772668/a1582435-7fde-4c82-8d5d-1dd7e3cfb2c7">

kubectl logs -f archery-5575cc8d47-qb4wz --tail=100 -n archery
Error from server (BadRequest): container "archery" in pod "archery-5575cc8d47-qb4wz" is waiting to start: PodInitializing

**Comment by LeoQuote** [comment]
PodInitializing 错误通常表示容器正在初始化，可能由于镜像拉取或其他初始化操作导致。您可以尝试以下方法来解决这个问题：

1. 检查 Pod 的描述，确保 Pod 中定义的容器名称和镜像名称正确。
2. 检查 Pod 的事件日志，查看是否有任何错误消息。
3. 检查 Pod 的状态和容器状态，确定具体哪个容器出现了问题。
4. 检查 Pod 的网络配置，确保容器能够正常访问所需的资源。
5. 尝试删除并重新创建 Pod，看看问题是否得到解决。

如果以上方法无法解决问题，您可以尝试查看更详细的日志信息或者尝试重启 Kubernetes 集群来解决问题。

**Reply by cslingjun** [reply]
![image](https://github.com/hhyo/Archery/assets/24772668/cf2ea4c0-fa84-4a82-ac70-2f16789a4284)
有个容器起不来，也没有日志，这个要如何排查呢 ，求指导

**Reply by cslingjun** [reply, marked as answer]
找到原因了 。
![image](https://github.com/hhyo/Archery/assets/24772668/f160cb93-f56f-4425-b398-78444a50f47e)
补充完整地址就好了 。https://github.com/hhyo/Archery/issues/2067
